### PR TITLE
Add documentation of CLI commands

### PR DESCRIPTION
**Pull request by RCheesley**
I am often finding I have to search for these so have pulled all the commands I could see as being relevant to a Mautic user into one table.

I didn't include all the doctrine commands etc as I think that isn't so relevant to the end-user, and I included at the start how to find them all if needed.

Fingers crossed the formatting looks half-decent!

### CODE REVIEW

**Comment by RCheesley** [comment]
Ah OK, should I remove them then?

**Comment by RCheesley** [comment]
> The transifex commands are not useful for an ordinary user. They are used by Transifex admins during the release process. But over all 👍

Just made a commit to remove them :) Thanks for the feedback!

File: en/tips/command-line.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+| mautic:plugins:update  | Installs, updates, enable and/or disable plugins.  |
+| mautic:queue:process   | Process queues  |
+| mautic:reports:scheduler  | Processes scheduler for report's export  |
+| mautic:segments:check-builders  | Compare output of query builders for given segments  |
+| mautic:segments:rebuild  | Update contacts in smart segments based on new contact data.  |
+| mautic:segments:update   | Update contacts in smart segments based on new contact data.  |
+| mautic:social:monitoring  | Looks at the records of monitors and iterates through them.   |
+| mautic:theme:json-config   | Converts theme config to JSON from PHP  |
+| mautic:transifex:pull  | Fetches translations for Mautic from Transifex  |
+| mautic:transifex:push   | Pushes Mautic translation resources to Transifex  |
```
Comment (escopecz) [inline en/tips/command-line.md]: I meant these transifex commands. I don't even know what the translation commands do 😄

Comment (RCheesley) [reply en/tips/command-line.md]: I just removed those two and the others relating to translation!

**Pull request merged**